ta/pkcs11/include/pkcs11_ta.h

```diff
@@ -565,8 +565,87 @@ enum pkcs11_ta_cmd {
 	 * This command relates to the PKCS#11 API function C_GenerateRandom().
 	 */
 	PKCS11_CMD_GENERATE_RANDOM = 42,
+
+	/*
+	 * PKCS11_CMD_DIGEST_INIT - Initialize a digest computation processing
+	 *
+	 * [in]  memref[0] = [
+	 *              32bit session handle,
+	 *              (struct pkcs11_attribute_head)mechanism +
+	 *                                            mechanism params,
```

Comment:
Remove the coma 
Ditto for other last arg in the inline description below.
That's a minor issue but lets get it fixed now :)

Reply:
OK

Reply:
By the way, I would prefer we use the short 1 line description already used in above commands
```
	 *              (struct pkcs11_attribute_head)mechanism + mecha params,
```
@ruchi393, would you mind if I post a P-R to change PKCS11_CMD_{SIGN|VERIFY}_INIT the same way?

Reply:
@etienne-lms , no issues. Please go ahead.